Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py

```diff
@@ -16,19 +23,22 @@ def csv_file_to_indicator_list(file_path, col_num, starting_row, auto_detect, de
     with open(file_path) as csv_file:
         file_reader = csv.reader(csv_file)
         for row in file_reader:
-            if line_index >= starting_row:
+            if line_index >= starting_row + offset and len(row) != 0:
                 indicator = row[col_num]
 
                 indicator_type = detect_type(indicator)
 
-                if indicator_type is None:
-                    continue
+                if not auto_detect:
+                    indicator_type = default_type
 
                 if type_col:
                     indicator_type = row[int(type_col) - 1]
 
-                elif not auto_detect:
-                    indicator_type = default_type
+                if indicator_type is None:
+                    if default_type is None:
+                        continue
+                    else:
+                        indicator_type = default_type
```

Comment:
extract to another function `get_indicator_type`

Reply:
and maybe add lines 24-25 to it, cause it seems related

Reply:
done